bevy_widgets/bevy_numeric_field/examples/many_fields.rs

```diff
@@ -0,0 +1,121 @@
+//! Contains example for spawning many numeric fields
+
+use bevy::prelude::*;
+use bevy_numeric_field::*;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_plugins(DefaultNumericFieldPlugin)
+        .add_systems(Startup, setup)
+        .run();
+}
+
+fn setup(mut commands: Commands) {
+    commands.spawn(Camera2dBundle {
+        camera: Camera {
+            clear_color: ClearColorConfig::Custom(Color::srgb(
+                34.0 / 255.0,
+                34.0 / 255.0,
+                34.0 / 255.0,
+            )),
+            ..default()
+        },
+        ..default()
+    });
+
+    commands
+        .spawn(NodeBundle {
+            style: Style {
+                width: Val::Percent(100.0),
+                height: Val::Percent(100.0),
+                justify_content: JustifyContent::Center,
+                align_items: AlignItems::Center,
+                flex_direction: FlexDirection::Row,
+                display: Display::Flex,
+                ..Default::default()
```

Comment:
for consistency i was say choose `..Default::default()` or `..default()`

Reply:
I rewrote all to ..Default::default()